code/modules/research/xenoarchaeology/tools/tools_pickaxe_vr.dm

```diff
@@ -0,0 +1,126 @@
+
+////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// Excavation pickaxes - sorted in order of delicacy. Players will have to choose the right one for each part of excavation.
+
+/obj/item/weapon/pickaxe/brush
```

Comment:
...I forgot about that. That would actually be a better way of going about with this. Give me a second to correct this.